boot/systems.go

```diff
@@ -447,3 +448,38 @@ func DropRecoverySystem(dev Device, systemLabel string) error {
 	const expectReseal = true
 	return resealKeyToModeenv(dirs.GlobalRootDir, m, expectReseal)
 }
+
+// AddRecoveryCapableSystem records a given system as one that we can recover
+// from.
+func AddRecoveryCapableSystem(systemLabel string) error {
+	opts := &bootloader.Options{
+		// setup the recovery bootloader
+		Role: bootloader.RoleRecovery,
+	}
+	// TODO:UC20: seed may need to be switched to RW
+	bl, err := bootloader.Find(InitramfsUbuntuSeedDir, opts)
+	if err != nil {
+		return err
+	}
+	rbl, ok := bl.(bootloader.RecoveryAwareBootloader)
+	if !ok {
+		return nil
+	}
+	vars, err := rbl.GetBootVars("snapd_recovery_capable_systems")
```

Comment:
how different will this list be from good_recovery_systems in modeenv? wondering if we need new terminology or not